docker/grouper/Dockerfile

```diff
@@ -0,0 +1,132 @@
+FROM jeanblanchard/java:jdk-7
+MAINTAINER Dennis Roberts <dennis@cyverse.org>
+ARG git_commit=unknown
+LABEL org.iplantc.de.javabase.git-ref="$git_commit"
+
+# Copy the configuration files to temporary locations.
+RUN mkdir -p /tmp/configs/api /tmp/configs/ui /tmp/configs/ws
+COPY api/grouper.properties /tmp/configs/api
+COPY api/grouper.hibernate.properties /tmp/configs/api
+COPY api/grouper-loader.properties /tmp/configs/api
+COPY api/subject.properties /tmp/configs/api
+COPY api/log4j.properties /tmp/configs/api
+COPY ui/build.properties /tmp/configs/ui
+COPY ws/build.properties /tmp/configs/ws
+COPY ws/grouper-ws.properties /tmp/configs/ws
```

Comment:
Depending how much capability for reuse of individual files you foresee, it might be more efficient to just put `api`, `ui`, and `ws` into a folder and just `COPY` that one folder in. These layers are small enough that the reuse may not matter much -- usually the cache is more useful for things like installing packages anyway, which take some time and network traffic.